### PR TITLE
Fix a very slight grammatical issue

### LINKED ISSUE #99972: Very slight grammatical issue in documentation
### Tested versions

Issue is present in latest documentation of the Godot Engine, highly possible that it has existed in earlier versions.

### System information

Windows 10 - Latest Documentation on Godot Engine

### Issue description

There is one specific issue within the documentation, due to the unfortunate oversight of the individual who wrote [this](https://docs.godotengine.org/en/latest/classes/class_packedfloat64array.html#class-packedfloat64array-method-to-byte-array) specific section. You see, in this quote:
> [PackedByteArray](https://docs.godotengine.org/en/latest/classes/class_packedbytearray.html#class-packedbytearray) to_byte_array() const [🔗](https://docs.godotengine.org/en/latest/classes/class_packedfloat64array.html#class-packedfloat64array-method-to-byte-array)
>
> Returns a copy of the data converted to a [PackedByteArray](https://docs.godotengine.org/en/latest/classes/class_packedbytearray.html#class-packedbytearray), where each element have been encoded as 8 bytes.
>
> The size of the new array will be float64_array.size() * 8.

There has been a grave grammatical error within this statement:

> where each element have been encoded as 8 bytes.

In this statement, "have" is not appropriate. The correct word should be "has".

### Steps to reproduce

Go to this [link](https://docs.godotengine.org/en/latest/classes/class_packedfloat64array.html#class-packedfloat64array-method-to-byte-array). The issue is self-evident.

### Minimal reproduction project (MRP)

Refer to reproduction steps above.

### PR DESCRIPTION
**Pull request by wyvrtn**
Changed the incorrect verb "have" into its correct form, "has." Fixes #99972

DANG IT SOMEONE TOOK MY 100,000th contrib

### CODE REVIEW

**Review by clayjohn — APPROVED** [review]

**Review by AThousandShips — APPROVED** [review]

**Review by akien-mga — CHANGES_REQUESTED** [review]
The same typo exists in many other files with the same description, which should be fixed too:
```
doc/classes/PackedInt64Array.xml
190:                            Returns a copy of the data converted to a [PackedByteArray], where each element have been encoded as 8 bytes.

doc/classes/PackedInt32Array.xml
189:                            Returns a copy of the data converted to a [PackedByteArray], where each element have been encoded as 4 bytes.

doc/classes/PackedFloat32Array.xml
195:                            Returns a copy of the data converted to a [PackedByteArray], where each element have been encoded as 4 bytes.

doc/classes/PackedFloat64Array.xml
196:                            Returns a copy of the data converted to a [PackedByteArray], where each element have been encoded as 8 bytes.
```

And also spotted this one with a quick regex search:
```
doc/classes/SurfaceTool.xml
150:                            Generates a tangent vector for each vertex. Requires that each vertex have UVs and normals set already (see [method generate_normals]).
```

**Comment by wyvrtn** [comment]
On it sire

**Comment by akien-mga** [comment]
Looks good! To match our workflow, the commits would need to be squashed (see [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions).

I'm happy to do it myself if you're not familiar with this process and it looks overwhelming (it can be, initially :D).

**Comment by wyvrtn** [comment]
> Looks good! To match our workflow, the commits would need to be squashed (see [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions).
> 
> I'm happy to do it myself if you're not familiar with this process and it looks overwhelming (it can be, initially :D).

Sire, I've squashed the commits. First time I've ever used rebase in fact.

**Review by akien-mga — APPROVED** [review]
Looks great!

File: doc/classes/SurfaceTool.xml [hunk 1]
```diff
@@ -147,7 +147,7 @@
 		<method name="generate_tangents">
 			<return type="void" />
 			<description>
-				Generates a tangent vector for each vertex. Requires that each vertex have UVs and normals set already (see [method generate_normals]).
+				Generates a tangent vector for each vertex. Requires that each vertex has UVs and normals set already (see [method generate_normals]).
```
Comment (AThousandShips) [inline doc/classes/SurfaceTool.xml:150]:
```suggestion
				Generates a tangent vector for each vertex. Requires that each vertex already has UVs and normals set (see [method generate_normals]).
```
Just minor suggestion, I think this flows better

**Review by AThousandShips — APPROVED** [review]

**Review by akien-mga — APPROVED** [review]

**Pull request merged**